apps/web/client/src/app/project/[id]/_components/right-panel/chat-tab/chat-messages/message-content/tool-call-simple.tsx

```
@@ -166,6 +173,13 @@ export function ToolCallSimple({
                     } else {
                         return 'Writing todos';
                     }
+                case GLOB_TOOL_NAME:
+                    const params12 = toolInvocation.input as z.infer<typeof GLOB_TOOL_PARAMETERS>;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)